libs/state/src/lib/rx-state.service.ts

```diff
@@ -547,13 +547,16 @@ export class RxState<T extends object> implements OnDestroy, Subscribable<T> {
     obsOrObsWithSideEffect: Observable<S>,
     sideEffectFn?: (arg: S) => void
   ): void {
+    const sideEffect = obsOrObsWithSideEffect.pipe(catchError((e)=> {
+      return EMPTY
+    }))
```

Comment:
Nitty change suggestion
```suggestion
    const sideEffect = obsOrObsWithSideEffect.pipe(catchError(e=> EMPTY))
```